src/EdFi.Db.Deploy/EdFi.Db.Deploy.csproj

```diff
@@ -13,16 +12,16 @@
     <Description>Database deployment utility for the Ed-Fi ODS/API solution.</Description>
     <Copyright>Copyright © 2019 Ed-Fi Alliance, LLC and contributors</Copyright>
     <PackageId>EdFi.Db.Deploy</PackageId>
-    <PackageProjectUrl>https://github.com/Ed-Fi-Alliance/Ed-Fi-Databases/tree/master</PackageProjectUrl>
-    <PackageLicense>https://www.ed-fi.org/license</PackageLicense>
+    <PackageProjectUrl>https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Databases/tree/master</PackageProjectUrl>
+    <PackageLicenseFile>LICENSE.txt</PackageLicenseFile>
     <RequireLicenseAcceptance>false</RequireLicenseAcceptance>
     <PackageType>DotnetTool</PackageType>
     <PackAsTool>true</PackAsTool>
     <ToolCommandName>EdFi.Db.Deploy</ToolCommandName>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
-
   <ItemGroup>
+    <None Include="../../LICENSE.txt" Pack="true" PackagePath="LICENSE.txt" />
```

Comment:
this looks better but i would rather see the none in the other item group so we are consistent with the grouping of items

Reply:
yes . Fixed it . Thanks